pkg/indexmanagement/scripts.go

```diff
@@ -55,14 +66,21 @@ CMD=$(cat <<END
 from __future__ import print_function
 import json,sys
 r=json.load(sys.stdin)
-indices = [index for index in r if int(r[index]['settings']['index']['creation_date']) < $minAgeFromEpoc ]
-if "$writeIndex" in indices:
-  indices.remove("$writeIndex")
-for i in range(0, len(indices), 25):
-  print(','.join(indices[i:i+25]))
+try:
```

Comment:
@blockloop @syedriko   looking at the outcome of the changes, what is the advantage in this case to add try/except.  It seems evaluating the exit code in the  change that follows is the only think needed

Reply:
I don't know ruby well enough to tell if `int(r[index]['settings']['index']['creation_date'])` this can raise an exception, but I'm not really a fan of exception-driven development anyway so if it works without the try/except then I say remove it.

Reply:
@jcantrill My impression was that we didn't know what value of ``writeIndices`` caused the ``TypeError: 'int' object has no attribute '__getitem__'`` exception. So I though 
``
except Exception as e:
    print(r)
    sys.exit(e)
``
would help us capture the offending content and figure out what's wrong with it.